packages/forms/src/elements/text/text.tsx

```diff
@@ -31,7 +32,7 @@ const InputText: React.FC<InputTextProps> = ({
 				type="text"
 				width={width}
 				testId={testId}
-				label={label}
+				label={ariaLabel}
```

Comment:
if `label` and `ariaLabel` can't be used together, in cases where `label` is specified instead of `ariaLabel`, the `aria-label` of the inner `input` will be `undefined`.  Should it be this instead?
`label={ariaLabel ? ariaLabel : label}`

Reply:
The desired behaviour would be no `aria-label` attribute, which I think is what we get from an `undefined` result. We could use the value of `label` but it's completely unnecessary so just bloats the response by a few bytes.